odw/services/minio/scripts/dev.sh

```diff
@@ -99,6 +161,7 @@ start_minio() {
 
     if is_minio_running; then
         print_warning "MinIO is already running"
+        setup_minio_buckets  # Still ensure buckets exist
```

Comment:
Does this mean we can remove dev-seed.sh so we dont have to use brew & mc to setup the buckets? It would be simpler so `pnpm odw:dev:seed` is basically just calling `seed-data.py`

Reply:
Yes. I wanted to keep that around for now since we're essentially deciding whether we're take this approach.